volt_cur_thresholds:
  battery_volt:
    warn: 19.5
    thruster_cutoff: 18.5
    diff_warn: 1.5
  battery_current:
    warn: 30
    fuse: 35
  thruster_current:
    warn: 7
    fuse: 10
  five_volt:
    warn_voltage_min: 4.7
    warn_voltage_max: 5.7
    warn_current_max: 10.0
  twelve_volt:
    warn_voltage_min: 11
    warn_voltage_max: 13
    warn_current_max: 10.0
  balanced_volt:
    warn_voltage_min: 18.5
    warn_voltage_max: 22
device_thresholds:
  copro_mem_warn_percentage: 90
  temp_over_target_warn: 10

# Lifetime of expiring message before it is marked as stale
ros_message_lifetime: 3